### PR TITLE
Translate 2.3.0-preview1 post (ja)

### PR DESCRIPTION
**Pull request by ryoben**
Please review

### CODE REVIEW

File: ja/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+---
+
+この度、Ruby 2.3.0-preview1 をリリースすることができました。
+
+これは、Ruby 2.3.0 の最初のプレビュー版で、たくさんの新機能と改良が含まれています。
+
+[Frozen String Literal
+Pragma](https://bugs.ruby-lang.org/issues/11473) Ruby 2.1 から`"str".freeze`により、文字列オブジェクトの生成が最適化されています。
+Ruby 2.3 ではファイル内の全ての文字列リテラルを対象にする新しいマジックコメントとコマンドラインオプションが提供されます。
+また、デバッグ用の`-enable-frozen-string-literal-debug`オプションによって、`“can’t modify frozen String”`エラーが発生した箇所を知ることができます。
```
Comment (JuanitoFatas) [inline ja/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: -enable => --enable

**Comment by stomar** [comment]
@ryoben Please amend your commit for this tiny fix and force push your branch.

**Comment by ryoben** [comment]
oops!
@JuanitoFatas @stomar  thanks!

File: ja/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+[Frozen String Literal
+Pragma](https://bugs.ruby-lang.org/issues/11473) Ruby 2.1 から`"str".freeze`により、文字列オブジェクトの生成が最適化されています。
+Ruby 2.3 ではファイル内の全ての文字列リテラルを対象にする新しいマジックコメントとコマンドラインオプションが提供されます。
+また、デバッグ用の`--enable-frozen-string-literal-debug`オプションによって、`“can’t modify frozen String”`エラーが発生した箇所を知ることができます。
+
+[Safe navigation operator](https://bugs.ruby-lang.org/issues/11537) これは、C＃やGroovy、Swiftで既に実装されているもので、`obj&.foo`で`nil`をハンドリングできます。また、`Array#dig`や`Hash#dig`も追加されます。
+
+[did_you_mean gem がバンドルされます](https://bugs.ruby-lang.org/issues/11252)。このgemは、`NameError`や`NoMethodError`の候補を表示し、デバッグをサポートします。
+
+是非、Ruby 2.3.0-preview1 を試して、結果をフィードバックしてください。
```
Comment (JuanitoFatas) [inline ja/news/_posts/2015-11-11-ruby-2-3-0-preview1-released.md]: Did you translate the "enjoy" part? Previous translation:

```
Ruby 2.2.0-preview1 でのプログラミングをお楽しみください！
```

**Comment by ryoben** [comment]
Use https links

**Comment by hsbt** [comment]
@ryoben Sorry, late response. this translation look good.

**Comment by ryoben** [comment]
@hsbt :ok_hand:

**Pull request merged**